package.json

```diff
@@ -27,11 +27,13 @@
     "test:watch": "npm run test -- -w ./lib",
     "lint": "jshint lib/ --config .jshintrc && jshint test/ --config test/.jshintrc",
     "test:coverage": "istanbul cover _mocha -- --recursive 'test/**/*.js' --reporter spec --exit",
+    "test:coveralls": "npm run test:coverage && cat ./coverage/lcov.info | coveralls && rm -rf ./coverage",
     "watch": "watch 'npm test && npm run lint' ./lib ./test"
   },
   "devDependencies": {
     "jshint": "~2.9.6",
     "clear-require": "~1.0.1",
+    "coveralls": "^3.0.2",
```

Comment:
We try to avoid `^` usage in packages.json dependencies (we have had bad experience with it in the past). Could you change it to `~`, please?

Reply:
Fixed 881300f